resources/js/store/store-supporter-tag.tsx

```diff
@@ -223,16 +229,15 @@ export default class StoreSupporterTag extends React.Component<Props> {
 
   @action
   private readonly getUser = (username: string) => {
-    this.xhr = $.ajax({
-      data: { username },
-      dataType: 'json',
-      type: 'POST',
-      url: route('users.check-username-exists'),
-    });
+    this.xhr = apiLookup([username]);
```

Comment:
the gift link generated in user menu should be `@` prefixed as it was previously strictly username lookup

Reply:
or maybe just prefix it before sending the request